### ISSUE TITLE
Dynamic Probing

### ISSUE THREAD

**Issue by doratheexplorr** [prometheus/blackbox_exporter#1058]
Hi Folks,

I have  a specific requirement where I need to probe an endpoint using http_post_2xx module but I want to Probe this end point once every 30 minutes. 
If the probe is successful continue to probe for every 30 minutes but if the probe fails I want to  probe this end point for every 5 minutes till this endpoint is up. Post that continue to probe every 30 minutes.

To sum it up:
If probe_success:1 ==> Probe the endpoint for 30 minutes
if probe_success:0 ==> Probe the endpoint for every 5 minutes till probe_success:1

**Comment by SuperQ** [comment]
This is not how the blackbox_exporter works. The exporter does not have any internal scheduling. Probe intervals/triggering is entirely external (Prometheus scrape interval). All probes are 1:1 per scrape.

**Comment by doratheexplorr** [comment]
Thanks!
Is there a way to scrape via Prometheus dynamically?

**Comment by SuperQ** [comment]
No, Prometheus does not work that way. Prometheus probes are meant to be at at most every 2 minutes.
